examples/spark2/src/test/scala/org/apache/carbondata/examplesCI/RunExamples.scala

```diff
@@ -19,12 +19,12 @@ package org.apache.carbondata.examplesCI
 
 import org.apache.spark.sql.test.util.QueryTest
 import org.scalatest.BeforeAndAfterAll
-
 import org.apache.carbondata.examples._
 import org.apache.carbondata.core.constants.CarbonCommonConstants
 import org.apache.carbondata.core.util.CarbonProperties
 import org.apache.carbondata.examples.sdk.CarbonReaderExample
 import org.apache.carbondata.examples.sql.JavaCarbonSessionExample
+import org.apache.spark.sql.test.TestQueryExecutor
```

Comment:
please move o another group

Reply:
@xubo245 Yeah, I move to another group.TestCarbonSerDe is also updated as same as the way of RunExamples.scala.